tuples.md

```diff
@@ -0,0 +1,67 @@
+# Writing Assignment - Tuples
+
+So far you have seen how to use arrays for holding a set of values and then iterate through them, for example:
+
+```swift
+var distances: [Int] = [10, 15, 20]
+
+for distance in distances {
+  print(distance)
+}
+```
+
+Arrays are a useful method for collecting variables, but limited as the values they contain can only be of one type. In this exercise we will introduce another, more flexible content type, the tuple.
+
```

Comment:
Arrays can actually contain types that conform to the same protocol. For instance, declaring an array that contains any object allows you to have an array that contains a string and a bool. (Not that someone would actually do this in practice but it is possible).

Another issue here is that the problem can be presented much organically and with a little bit more "struggle." This situation might not be the right one, but for example after you present a concrete real-world example that uses an array can you pose a question such as "Now what if we wanted to print...." and that can be the jumping point to presenting new material. 

Or in another totally new example to start with, take a function that returns one thing and ground in a real world example.  And then ask the question of how you would have this function return two things? That's where you set the stage for tuples. 
- E.g We have a dictionary of movies and their directors. We want to print a sentence that takes a movie and its directors and says "[movie] was directed by [director]" How would we go about doing that? 
 -Now you can present the answer.